app/models/setting/content.rb

```diff
@@ -106,9 +106,9 @@ def self.default_settings
                "/var/lib/pulp/katello-export", N_('Pulp export destination filepath')),
       self.set('pulpcore_export_destination', N_("On-disk location for pulp 3 exported repositories"),
                "/var/lib/pulp/exports", N_('Pulp 3 export destination filepath')),
-      self.set('pulp_client_key', N_("Path for ssl key used for pulp server auth"),
+      self.set('pulp_client_key', N_("Path for ssl key used for pulp server auth (deprecated)"),
```

Comment:
AFAIK today you still need a DB migration to change descriptions (since descriptions are still stored in the DB, until https://community.theforeman.org/t/setting-in-memory-only-changed-values-in-database/18976 is done)

Reply:
That's a bummer. And I don't suppose we have a built in way to mark settings as deprecated?

Reply:
I don't know. Perhaps @ezr-ondrej knows.